content/04.discussion.md

```diff
@@ -1,3 +1,25 @@
 ## Discussion
 
-Stub in discussion section   
+Through OpenPBTA, we used an open, reproducible analysis framework within GitHub to bring together researchers and clinicians across the nation to genomically characterize 943 pediatric brain tumors across 59 distinct histologies.
+To our knowledge, this was the first large-scale, collaborative open analysis of genomic data coupled with open manuscript writing in which we comprehensively analyzed the largest cohort of pediatric brain tumors to date.
+We used available WGS, WXS, and RNA-Seq data to generate high-confidence consensus SNV and CNV calls, prioritize putative oncogenic fusions, and create over 40 code- and scientifically-reviewed scalable modules for common downstream cancer genomics analyses which have been used to create new knowledge as well as define new questions.
+We detect and show expected frequencies of genomic lesions, mutational signatures, and aberrantly regulated signaling pathways across multiple pediatric brain tumor histologies.
+Molecular subtyping information was largely not collected nor available for this cohort, and if available (e.g.: sparse medulloblastoma subtypes), it had to be manually curated from pathology reports and/or free text clinical data fields.
+Furthermore, this tumor cohort has yet to be profiled by DNA methylation. 
```

Comment:
Seems out of place. Does this mean the lack of methylation profiling is related to lack of subtyping? If so, rephrase to something like - "Furthermore, this tumor cohort has yet to be profiled by DNA methylation, increasing the challenge of determining molecular subtypes."

Reply:
oh yes - generally, brain tumors are profiled by methylation in order to determine subgroupings, and we did not have that for this project (underway currently, though!)

Reply:
fyi: https://www.molecularneuropathology.org/mnp/classifiers/11